src/dev/konnector.md

```diff
@@ -2,35 +2,52 @@
+## How does it work ?
 
-From a technical point of view, connectors are Node.js applications executed inside a container. They communicate with the server using its API, like client side apps, and get an auth token every time they start. They need to register with a manifest, and ask permissions to the user.
+A connector is a NodeJS script. The target node version used to run your connector is the
+[current LTS version](https://nodejs.org) (8 at the moment).
 
-To ease the development of a connector, an npm package, named [cozy-connector-libs](https://github.com/cozy/cozy-konnector-libs) provides a lot of
-shared libraries. But you may need some other npm packages to help you running your connector: 
+Like client side apps, connectors communicate with the [Cozy Stack](https://cozy.github.io/cozy-stack/)
+using its API, and get an auth token every time they start. They need to register with a manifest,
+and ask permissions to the user.
 
- - [moment](http://momentjs.com/docs/) to manage dates 
+To ease the development of a connector, a npm package, named [cozy-konnector-libs](https://github.com/cozy/cozy-konnector-libs)
+provides some shared libraries which are adapted to be used for a connector :
+ - [cheerio](https://cheerio.js.org) to easily request html pages like jQuery
+ - [request-promise](https://github.com/request/request-promise): [request](https://github.com/request/request) wrapped in promises
+ - [request-debug](https://github.com/request/request-debug) which displays all the requests and
+   responses in the standard output. Check "debug" option in [requestFactory](https://github.com/cozy/cozy-konnector-libs/blob/master/docs/api.md#requestFactory)
 
-When the application is started, it also gets some data through environment variables:
+But you may need some other npm packages to help you running your connector and
+which are not integrated to cozy-konnector-libs :
+
+ - [momentjs](http://momentjs.com/docs/) or [dade-fns](https://date-fns.org) to manage dates
+ - [bluebird](http://bluebirdjs.com) to get enhanced promises
+
+When the connector is started, it also gets some data through environment variables:
 
  - `COZY_CREDENTIALS` : the auth token used by `cozy-client-js` to communicate with the server 
  - `COZY_URL` : the API entry point 
- - `COZY_FIELD` : the settings specific to each connector, for example the path of folder where the user wants to store the remote files 
-
-But the base connector (`require('cozy-konnector-libs').BaseKonnector`) in cozy-konnector-libs handles these for you.
+ - `COZY_FIELDS` : the settings comming from [cozy-collect](https://github.com/cozy/cozy-collect) and filled by the user of the connector (login, password, directory path).
 
-The application can access a temporary file system, deleted at the end of its execution. Its logs (standard and error output) are kept by the server.
+But the BaseKonnector (`require('cozy-konnector-libs').BaseKonnector`) in cozy-konnector-libs handles
```

Comment:
d/But/

Those variables are used by the BaseKonnector and the cozy-client to configure the connection to the stack with the right permissions as defined in your `manifest.konnector`.